soda/sqlserver/soda/data_sources/sqlserver_data_source.py

```diff
@@ -129,6 +130,7 @@ def handle_datetimeoffset(dto_value):
                 ("Trusted_Connection=YES;" if self.trusted_connection else "")
                 + ("TrustServerCertificate=YES;" if self.trust_server_certificate else "")
                 + ("Encrypt=YES;" if self.encrypt else "")
+                + ("MultiSubnetFailover=YES;" if self.multi_subnet_failover else "")
```

Comment:
I can try to add the bare minimum, but I won't focus on a full generic system right now, especially since we might end up needing a custom method for each data source.

Reply:
Please check the proposed solution.